tools/scripts/publishDocker.mjs

```diff
@@ -0,0 +1,22 @@
+import { getReleaseTag, getVersionByArguments } from './version.mjs';
+import path from 'path';
+import fs from 'fs';
+import { execSync } from 'child_process';
+
+if (process.env.READY_FOR_PUBLISH !== 'true') {
+  console.log(`Set env READY_FOR_PUBLISH=true before running publish commands.`)
+  process.exit(1);
+}
+
+const packageName = JSON.parse(fs.readFileSync(path.resolve(process.cwd(), 'package.json'), 'utf-8')).name.replace('@vulcan-sql/', '');
+if (!packageName) throw new Error(`Can't find package name`);
+const version = getVersionByArguments();
+const tag = getReleaseTag();
+
+// run docker build xxx -t vulcan-sql/<package-name> before publish
+// we tag it with url and version
+execSync(`docker tag vulcan-sql/${packageName} ghcr.io/canner/vulcan/${packageName}:${version}`);
+execSync(`docker tag vulcan-sql/${packageName} ghcr.io/canner/vulcan/${packageName}:${tag}`);
+// push
+execSync(`docker push ghcr.io/canner/vulcan/${packageName}:${version}`);
+execSync(`docker push ghcr.io/canner/vulcan/${packageName}:${tag}`);
```

Comment:
Why do you make the docker image have different tag by `version` and `tag` ? Is it any case that could be needed? 

Because only follow the code, since the `getReleaseTag` function, looks like it will not related to `version` in `package.json`, but we allow user could tag one of the `dev`, `beta`, `lastest`, so usually user will also add the version before or after the tag e.g: **dev-0.1.0**, **0.1.0-dev**, but when the `version` in the `package.json` is `0.0.9` it will make it a little confusing.

Reply:
> Why do you make the docker image have different tag by `version` and `tag` ? Is it any case that could be needed?

I'd like to move `dev`, `beta`, and `latest` tags to the latest image we pushed, for example:
Today
0.1.0-dev.20220907.0 dev
Tomorrow
0.1.0-dev.20220907.0
0.1.0-dev.20220908.0 dev


> Because only follow the code, since the `getReleaseTag` function, looks like it will not related to `version` in `package.json`, but we allow user could tag one of the `dev`, `beta`, `lastest`, so usually user will also add the version before or after the tag e.g: **dev-0.1.0**, **0.1.0-dev**, but when the `version` in the `package.json` is `0.0.9` it will make it a little confusing.

I should update package.json with the desired version before building, I'll fix it.